### ISSUE TITLE
[TIARA] Entity Linker.

### ISSUE THREAD

**Issue by PrayushiFaldu** [microsoft/KC#28]
Hello, it is mentioned in the readme file to use the PURE project for training mention detection model.
Command:  python run_entity.py \
    --do_train --do_eval --eval_test \
    --learning_rate=5e-6 --task_learning_rate=5e-6 \
    --train_batch_size=32 \
    --eval_batch_size=108 \
    --context_window 0 \
    --max_span_length 15 \
    --task grailqa \
    --data_dir grailqa_data/json/ \
    --model bert-base-uncased \
    --output_dir grailqa_models/checkpoint \
    --num_epoch 10 \
    --seed 42

But the PURE project has no task "grailqa"  as given in arguments.
Can you also mention the code changes required in PURE to make it run for grailqa.
